.github/workflows/test_dbt_models.yaml

```diff
@@ -29,6 +30,20 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Validate and parse input variables
+        id: parse-inputs
+        run: |
+          # Default to running all tests
+          SELECT_OPTION="--selector select_data_test_iasworld"
+          if [[ -n "$SELECT" ]]; then
+            SELECT_OPTION="--select $SELECT"
+          fi
+          echo "Setting select option to '$SELECT_OPTION'"
+          echo "select-option=$SELECT_OPTION" >> "$GITHUB_OUTPUT"
```

Comment:
**question (non-blocking):** Is GitHub okay with hyphenated env vars/outputs? This definitely wouldn't work in bash.